torchao/float8/fsdp_utils.py

```diff
@@ -258,6 +261,16 @@ def fsdp_post_all_gather(
 
 
 class WeightWithDelayedFloat8CastTensor(torch.Tensor):
+
+    __slots__ = [
```

Comment:
curious is `__slots__` derived from `__tensor_flatten__` ?

Reply:
Yes, here I just collected all attributes used in __tensor_flatten__, __tensor_unflatten__